Essentials/src/com/earth2me/essentials/commands/Commandspeed.java

```diff
@@ -133,18 +131,9 @@ private float getRealMoveSpeed(final float userSpeed, final boolean isFly, final
         }
     }
 
-    @Override
-    protected List<String> getTabCompleteOptions(Server server, User user, String commandLabel, String[] args) {
-        if (args.length == 3 && user.isAuthorized("essentials.speed.others")) {
-            return getPlayers(server, user);
-        } else {
-            return Collections.emptyList();
-        }
-    }
-
     @Override
     protected List<String> getTabCompleteOptions(Server server, CommandSource sender, String commandLabel, String[] args) {
```

Comment:
This tab method is a little weird, because it doesn't actually suggest anything until the user has entered 2 args already, making it fairly useless currently.